syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.pwave";
option java_outer_classname = "Pwave";
option objc_class_prefix = "PW";

package pwave;

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}


// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}


message ray_t {
  string test = 1;
}

message ray_config_t {
  string test = 1;
}

service Raytrace {

  rpc compute(ray_config_t) returns (ray_t);

};
